stdlib/public/core/UTF8Span.swift

```diff
@@ -190,7 +199,10 @@ extension UTF8Span {
 }
 
 extension String {
-
+  /// Create a new String instance, copying over the contents of `codeUnits`.
+  /// This initializer is able to skip UTF-8 validation, as `codeUnits` must contain valid UTF-8.
```

Comment:
```suggestion
  /// Creates a new string, copying the specified code units.
  ///
  /// This initializer skips UTF-8 validation because `codeUnits` must contain valid UTF-8.
```

Abstract is always exactly one sentence, and the blank line separates additional discussion from the abstract.  Don't refer to parameters by name in the abstract, because the abstract needs to be readable in other contexts, like navigation ad Xcode quick help, where the parameters don't appear alongside it.

Changed "is able to" to "skips" because it sounds like that validation is always skipped -- is this correct?

I think "because" here is clearer than "as" (but Apple Style Guide doesn't explicitly forbid what you had.)